admin/views/workouts.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Views
+ *
+ * @uses string $cornerstone_guide  The link to the cornerstone guide.
+ * @uses string $cornerstone_upsell The link to buy premium from the cornerstone workout.
+ */
+
+if ( ! defined( 'WPSEO_VERSION' ) ) {
+	header( 'Status: 403 Forbidden' );
+	header( 'HTTP/1.1 403 Forbidden' );
+	exit();
+}
+
+?>
+
+<div id='wpseo-workouts-container'>
+	<div>
+		<h1>
+			<?php esc_attr_e( 'SEO Workouts', 'wordpress-seo' ); ?>
+		</h1>
+		<p>
+			<?php
+			esc_attr_e(
+				'Getting your site in shape and keeping it SEO fit can be challenging. Let us help you get started by taking on the most common SEO challenges, with these step by step SEO workouts.',
+				'wordpress-seo'
+			);
+			?>
+		</p>
+		<div class='card'>
+			<h2><?php esc_attr_e( 'The cornerstone approach', 'wordpress-seo' ); ?></h2>
```

Comment:
```suggestion
			<h2><?php esc_html_e( 'The cornerstone approach', 'wordpress-seo' ); ?></h2>
```

`esc_attr` is used for escaping here which is intended for attributes, `esc_html` and it's variants should be used instead.